native/src/main/java/org/ballerinax/asb/receiver/MessageReceiver.java

```diff
@@ -407,53 +448,62 @@ public static Object receiveDeferred(BObject receiverClient, Object sequenceNumb
      * @param message        Message object.
      * @return An error if failed to renewLock of the message.
      */
-    public static Object renewLock(BObject receiverClient, BMap<BString, Object> message) {
-        try {
-            ServiceBusReceiverClient receiver = getReceiverFromBObject(receiverClient);
-            ServiceBusReceivedMessage nativeMessage = getNativeMessage(message);
-            receiver.renewMessageLock(nativeMessage);
-            LOGGER.debug(String.format("Done renewing a message(Id: %s) using its lock token from %s",
-                    nativeMessage.getMessageId(), receiver.getEntityPath()));
-            return null;
-        } catch (BError e) {
-            return ASBErrorCreator.fromBError(e);
-        } catch (ServiceBusException e) {
-            return ASBErrorCreator.fromASBException(e);
-        } catch (Exception e) {
-            return ASBErrorCreator.fromUnhandledException(e);
-        }
+    public static Object renewLock(Environment env, BObject receiverClient, BMap<BString, Object> message) {
+        ServiceBusReceivedMessage nativeMessage = getNativeMessage(message);
+        ServiceBusReceiverClient receiver = getReceiverFromBObject(receiverClient, false);
```

Comment:
```suggestion
        ServiceBusReceiverClient receiver = getReceiverFromBObject(receiverClient);
```

Reply:
lets update all the similar places

Reply:
Related to this comment: 
- https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/230#discussion_r1630575178

Reply:
+1